scripts/hook.py

```diff
@@ -149,6 +155,25 @@ def forward(self, x, timesteps=None, context=None, **kwargs):
                     cond = torch.cat([cond, total_extra_cond], dim=1)
                     uncond = torch.cat([uncond, uncond[:, -total_extra_cond.shape[1]:, :]], dim=1)
                     context = torch.cat([cond, uncond], dim=0)
+            elif (len(total_extra_cond) > 0
+            and   outer.stylebreak):
+                # SBM Stylebreak fix: Context is split to (cond,uncond) iterations.
+                # Since I don't receieve an indication of which iteration is which,
+                # I rely instead on a simple counter and assumption that the order is maintained.
+                
+                # SBM Perhaps a simple reshape(1,*total.shape) would do.
+                total_extra_cond = torch.repeat_interleave(total_extra_cond.unsqueeze(0), context.shape[0], dim=0)
+                indpos = True
+                if outer.is_vanilla_samplers: # Vanillas reverse cond,uncond order.
+                    indpos = not indpos
+                if outer.cntiter % 2 != 0: # Flip cond/uncond in odd iters. 
```

Comment:
I suspect this is not going to work well with multidiffusion. Some extensions double, triple, quadruple the number of passes sometimes per prompt.

Reply:
You're probably right about that, but I don't understand the flow well enough to be able to pass a more direct indication.
At the least, it should still work if pos/neg are called intermittently for every pass.

Reply:
Apologies for the delay. It seems to be working with multidiffusion after a bit of testing.